src/views/expensesgroup/addexpense/Dialogs/AddExpenseDialog.js

```diff
@@ -23,7 +29,23 @@ function AddExpenseDialog(props) {
                     <CloseIcon />
                 </IconButton>
             </DialogTitle>
-            <DialogContent sx={{ width: '490px' }}>bbb</DialogContent>
+            <DialogContent>
+                <Typography variant="body2" sx={{ fontWeight: 600 }}>
+                    Description
+                </Typography>
+                <TextField type="text" name="quantity" fullWidth />
+                <Typography variant="body2" sx={{ mt: '32px', fontWeight: 600 }}>
+                    Amount
+                </Typography>
+                <TextField type="number" name="quantity" fullWidth />
+                <Typography variant="body2" sx={{ mt: '32px', fontWeight: 600 }}>
+                    Budget
+                </Typography>
+                <TextField type="text" name="quantity" fullWidth />
```

Comment:
With the change that I mentioned you can now know which budget the user is trying to add.
So remove the TextField  and show it like this:

![image](https://user-images.githubusercontent.com/35969631/160107226-62e33505-63f3-443d-96f0-53f0fe8638aa.png)